src/extension/background-script/events/persistPayments.js

```diff
@@ -1,16 +1,16 @@
 import db from "../db";
 
 const persistSuccessfullPayment = async (message, data) => {
-  const recipientName = data.origin.name;
-  const host = data.origin.host;
+  const host = data?.origin?.host || "";
```

Comment:
do we want the host to be an empty string? is this a good practice or should it rather be something null? 
I assume the empty string is to make TS happy but wouldn't a null value be more useful in a DB?